### PR TITLE
übersetzung ins deutsche

### PR DESCRIPTION
**Pull request by sushi2k**
Thank you for submitting a Pull Request to the Mobile Application Security Verification Standard. Please make sure that:

- [ ] Your PR is linked to an issue.
- [ ] Your PR follows the [contribution guidelines](https://github.com/OWASP/owasp-masvs/blob/master/CONTRIBUTING.md "Contribution guidelines")
- [ ] All links are valid and use HTTPS and have been formatted as [TEXT](URL “NAME”)
- [ ] In case of a translation: have you made sure that the translation from EN is as close to the EN document as possible?

This PR covers issue #242 .

If your pull-request has a focus of rewriting a requirement and/or adding a new one, please make sure you have a backing issue in which your change has been discussed and agreed. If there is none, then your PR will be closed without further review.

### CODE REVIEW

File: Document-de/CHANGELOG.md [hunk 1]
```diff
@@ -1,57 +1,58 @@
...
-- Added a Code of Conduct & Contributing guidelines.
-- Added a Pull-Request template.
-- Updated the sync with the repo in use for hosting the Gitbook website.
-- Updated the scripts to generate XML/JSON/CSV for all the translations.
-- Translated the Foreword to Chinese (ZHTW).
+- Markdown Fehler wurden alle behoben.
+- Französische und Spanische Übersetzung wurden aktualisiert.
+- Versionshistorie wurde übbersetzt in Chinesisch (ZHTW) und Japanisch.
+- Automatische überprüfung der Markdown Syntax und Erreichbarkeit von URLs.
+- IDs zur identifizierung von Anforderungen wurden hinzugefügt, die in der nächsten Veröffentlichung des MSTG mit eingearbeitet werden. Dies soll die Suche nach Anforderungen und Testfällen vereinfachen.
```
[the remark this answers is not in the document]
Comment (sushi2k) [reply Document-de/CHANGELOG.md]: 👍

File: Document-de/CHANGELOG.md [hunk 2]
```diff
@@ -1,57 +1,58 @@
...
-## V1.1.0 14 July 2018
+## V1.1.0 14 Juli 2018
 
-The following changes are part of release 1.1:
+Die folgenden Änderungen sind Teil von Version 1.1:
 
-- Requirement 2.6 "The clipboard is deactivated on text fields that may contain sensitive data." was removed.
-- Requirement 2.2 "No sensitive data should be stored outside of the app container or system credential storage facilities." was added.
-- Requirement 2.1 was reworded to "System credential storage facilities are used appropriately to store sensitive data, such as PII, user credentials or cryptographic keys.".
+- Anforderung 2.6 "The clipboard is deactivated on text fields that may contain sensitive data." wurde entfernt.
```
[the remark this answers is not in the document]
Comment (sushi2k) [reply Document-de/CHANGELOG.md]: Damals gab es noch keine Deutsche Übersetzung und da es entfernt wurde, wurde auch nie übersetzt.

**Comment by sushi2k** [comment]
Danke @codethatrocks ! Alles gefixt und gepusht. Wenn alles passt, kannst du den PR dann bitte approven und mergen?

**Comment by codethatrocks** [comment]
Hallo Sven,

approved ist. Für den merge fehlen mir die rechte.

Viele Grüße
Rocco

**Pull request merged**